dns.go

```diff
@@ -48,11 +48,29 @@ type DNSRecord struct {
 	Prio       int    `json:"prio,omitempty"`
 }
 
+// DNSRecordUpdate represents a DNS record
+type DNSRecordUpdate struct {
```

Comment:
to follow the CloudStack logic, I'd move the verb at the beginning: `CreateDNSRecord`, `UpdateDNSRecord`

Reply:
oh yes ! thx